src/libraries/System.Private.CoreLib/src/System/Number.Formatting.cs

```diff
@@ -1297,6 +1297,7 @@ private static unsafe bool TryInt32ToHexStr(int value, char hexBase, int digits,
             return true;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
We're ok with whatever additional code size this leads to? (General question for the AggressiveInlinings added in the PR)